src/Exception/InvocationFailureException.php

```diff
@@ -0,0 +1,89 @@
+<?php
+
+namespace Dhii\Invocation\Exception;
+
+use Exception as RootException;
+use Dhii\Invocation\CommandInvokerInterface;
+use Dhii\Invocation\ArgsAwareTrait;
+use Dhii\Invocation\CommandAwareTrait;
+use Dhii\Util\Normalization\NormalizeArrayCapableTrait;
+
+/**
+ * An exception that occurs in relation to a command invoker.
+ *
+ * @since [*next-version*]
+ */
+class InvocationFailureException extends AbstractBaseCommandInvokerException implements InvocationFailureExceptionInterface
+{
+    /*
```

Comment:
Consider using `NormalizeStringCapableTrait` and `NormalizeIntCapableTrait` here instead of in `AbstractBaseCommandInvokerException`.